sentry-android-ndk/src/main/java/io/sentry/android/ndk/SentryNdk.java

```diff
@@ -26,5 +27,10 @@ public static void init(@NotNull final SentryOptions options) {
     SentryNdkUtil.addPackage(options.getSdkVersion());
     initSentryNative(options);
     options.addScopeObserver(new NdkScopeObserver(options));
+
+    if (options instanceof SentryAndroidOptions) {
```

Comment:
`m`: Shouldn't the `SentryNdk.init` always receive `SentryAndroidOptions`? Can we change this?

Reply:
yeah I didn't want to couple them but it's the right thing to do, will do it.